### PR TITLE
Analytic units' panel fields are not saved to db #523

### LINKED ISSUE #523: Analytic units' panel fields are not saved to db
Fields used only by panel (e.g. `labeledColor`, `detectorType`, etc) are not saved to db
Affects:
https://github.com/hastic/hastic-grafana-app/issues/233
https://github.com/hastic/hastic-grafana-app/issues/232

### PR DESCRIPTION
**Pull request by rozetko**
Fixes #523 

Changes:
- Make `toObject()` `AnalyticUnit`'s method return all the fields

### CODE REVIEW

File: server/src/models/analytic_unit_model.ts [hunk 1]
```diff
@@ -115,7 +115,11 @@ export class AnalyticUnit {
       alert: this.alert,
       lastDetectionTime: this.lastDetectionTime,
       status: this.status,
-      error: this.error
+      error: this.error,
+      labeledColor: this.labeledColor,
```
Comment (jonyrock) [inline server/src/models/analytic_unit_model.ts]: can we use "positiveLabelColor"

File: server/src/models/analytic_unit_model.ts [hunk 2]
```diff
@@ -115,7 +115,11 @@ export class AnalyticUnit {
...
+      deletedColor: this.deletedColor,
```
Comment (jonyrock) [inline server/src/models/analytic_unit_model.ts]: and "negativeLabelColor"

File: server/src/controllers/analytics_controller.ts [hunk 3]
```diff
@@ -199,6 +199,7 @@ export async function runLearning(id: AnalyticUnit.AnalyticUnitId) {
       await AnalyticUnitCache.create(id);
     }
 
+    // TODO: create analytics serialization method in AnalyticUnit
```
Comment (jonyrock) [inline server/src/controllers/analytics_controller.ts]:
```suggestion
    // TODO: create an analytics serialization method in AnalyticUnit
```

File: server/src/models/analytic_unit_model.ts (hunk 1 above)
Comment (rozetko) [reply server/src/models/analytic_unit_model.ts]: Let's do it in the separate PR cause it involves DB migration

**Pull request merged**